package.json

```diff
@@ -14,6 +14,7 @@
   "author": "ambanum",
   "type": "module",
   "exports": {
+    ".": "./src/exports.js",
     "./fetch": "./src/archivist/fetcher/exports.js",
     "./filter": "./src/archivist/filter/exports.js",
     "./page-declaration": "./src/archivist/services/pageDeclaration.js"
```

Comment:
Is this still needed when we now list exports in `exports.js`?

Reply:
Yes, it allows to import in two different ways:
- `import { Fetch } from 'opentermsarchive';`
- `import Fetch from 'opentermsarchive/fetch';`